features/kinderen.feature

```diff
@@ -164,10 +172,7 @@ Functionaliteit: Kinderen van een persoon raadplegen
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550005                       |
       | fields              | burgerservicenummer,kinderen    |
-      Dan heeft de persoon met burgerservicenummer '555550005' een 'kind' met alleen de volgende gegevens
-      | naam              | waarde       |
-      | type              | KindOnbekend |
-      | indicatieOnbekend | true         |
+      Dan heeft de persoon met burgerservicenummer '555550005' een 'kind' zonder gegevens
```

Comment:
dit klopt niet, want de geboortedatum wordt geleverd als type DatumOnbekend.

Je zou de fields moeten aanscherpen tot bijvoorbeeld burgerservicenummer,kinderen.naam,kinderen.geboorte.plaats

Reply:
Aangepastr voorstel overgenomen

Reply:
Als ik er nu nog eens naar kijk is deze rule "Wanneer een kind alleen "onbekend"-waardes heeft, wordt er een 'kind' zonder gegevens geleverd." en het scenario niet correct. Althans niet in lijn met wat ik beschrijf in lege-waarden.feature in P.R. #1129.

fields=burgerservicenummer,kinderen.naam,kinderen.geboorte.plaats
en in de bron is burgerservicenummer leeg, naam.geslachtsnaam="." en geboortedatum="00000000".
De conclusie "Dan heeft de persoon met burgerservicenummer '555550005' een 'kind' zonder gegevens" is dan denk ik niet beschrijvend genoeg.

Ik verwacht namelijk als resultaat:
"kinderen" [{ "naam": {}, "geboorte": {} }]

Dat is de consequentie van consequent toepassen van de regel dat een object dat wel bestaat maar geen te tonen waarde heeft, moet worden opgenomen als leeg object. In dit geval bestaan objecten naam en geboorte, alleen zijn er geen te tonen waarden. Je moet beide dus als leeg object leveren en niet weglaten.

Reply:
Deze opmerking is opgevoerd in #1155 en wordt buiten deze PR opgepakt.